blog/content/post/2023-04-27-3mdeb-open-source-contribution.md

```diff
@@ -0,0 +1,225 @@
+---
+title: "3mdeb Insights: A Look into 3mdeb's Open-Source Contributions"
+abstract: "We're thrilled to share our team's remarkable open-source
+           contributions from 2021 to the first half of 2023 with you.
+           We've actively worked on enhancing various projects, including
+           coreboot and fwupd, leaving a significant impact on the industry.
+           Our involvement has not only boosted the functionality and security
+           of these projects but also equipped us with valuable expertise for
+           our clients. Stay tuned for an insightful blog post diving into our
+           open-source contributions!"
```

Comment:
Please rewrite based on following example: https://github.com/3mdeb/news-and-ideas/blob/master/blog/content/post/2023-05-18-our_contribution_to_coreboot_4_20_release.md?plain=1#L3